dask/dataframe/tests/test_rolling.py

```diff
@@ -8,11 +8,11 @@
 from dask.utils import raises, ignoring
 
 
-def eq(p, d):
+def eq(p, d, check_less_precise=False):
     if isinstance(d, dd.DataFrame):
-        tm.assert_frame_equal(p, d.compute(get=get_sync))
+        tm.assert_frame_equal(p, d.compute(get=get_sync), check_less_precise=check_less_precise)
     else:
-        tm.assert_series_equal(p, d.compute(get=get_sync))
+        tm.assert_series_equal(p, d.compute(get=get_sync), check_less_precise=check_less_precise)
```

Comment:
Oh, we should consider using the `eq` function in `dask.dataframe.utils` which checks a lot more and already passes this through.